python/tests/server/test_response_throttler.py

```diff
@@ -0,0 +1,41 @@
+import time
+
+from cog.server.response_throttler import ResponseThrottler
+from cog.response import Status
+
+def test_zero_iterval():
+    throttler = ResponseThrottler()
+    throttler.response_interval = 0
+
+    assert throttler.should_send_response({"status": Status.PROCESSING})
+    throttler.update_last_sent_response_time()
+    assert throttler.should_send_response({"status": Status.SUCCEEDED})
+
+
+def test_terminal_status():
+    throttler = ResponseThrottler()
+    throttler.response_interval = 10
+
+    assert throttler.should_send_response({"status": Status.PROCESSING})
+    throttler.update_last_sent_response_time()
+    assert not throttler.should_send_response({"status": Status.PROCESSING})
+    throttler.update_last_sent_response_time()
```

Comment:
I don't think we need this line

Reply:
I added those to make the test work like the class is used in practice. But no strong opinions.